packages/core/src/api/meta/initialize.ts

```diff
@@ -0,0 +1,15 @@
+import type { Meta, Metas, User } from '../../metas';
+import type { Transports } from '../../transports';
+import type { MetaAPI } from './types';
+
+export function initializeMeta(_: Transports, metas: Metas): MetaAPI {
+  let meta: Meta = {};
+
+  metas.add(() => meta);
+
+  return {
+    setUser: (user: User | null) => {
```

Comment:
Should we move `setUser` into a separate constant and just reference it here? It would be aligned w/ the rest of the agent code. A benefit would be easier maintenance on the long run.

Reply:
- [ ] refac